### PR TITLE
Leaderboard: Add leaderboard structure

### PR DESCRIPTION
**Pull request by Emmy-github-webdev**
# Leaderboard: Add leaderboard structure

In leaderboard project I:
- Setup a linter
- Setup Webpack
- Implement Gitflow
- Write plain HTML markup for leaderboard

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/* eslint-disable import/no-cycle */
+import './css/styles.css';
+import _ from 'lodash';
+
+function component() {
+  const element = document.createElement('div');
```
Comment (henatan99) [inline src/index.js]: - [ ] Kindly use the ES6 syntax for function definition (arrow function), as one of the objectives of this project is to practice ES6 syntax. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @Emmy-github-webdev , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [ ] A professional repo with descriptive readme is one of the general requirements of the project. Kindly update your readme with minimal information using the Microverse [readme template](https://github.com/microverseinc/readme-template). :pray: 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @Emmy-github-webdev , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Emmy-github-webdev** [comment]
Thank you

File: src/index.js (hunk 1 above)
Comment (Emmy-github-webdev) [reply src/index.js]: Recommendations implemented. Thank you

**Pull request merged**